docs/book/src/cronjob-tutorial/testdata/project/internal/controller/cronjob_controller_test.go

```diff
@@ -29,8 +29,8 @@ import (
 	"reflect"
 	"time"
 
-    . "github.com/onsi/ginkgo/v2"
-    . "github.com/onsi/gomega"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
```

Comment:
hi @hsadoyan 

See: https://github.com/kubernetes-sigs/kubebuilder/actions/runs/9996175604/job/27630173536?pr=4016
It is failing in the testdata
So, you need to:
- rebase your branch with master
- then, run make generate locally. This target will update all places and docs with the changes.